cli/azd/pkg/auth/manager.go

```diff
@@ -86,9 +88,10 @@ func NewManager(configManager config.UserConfigManager) (*Manager, error) {
 		return nil, fmt.Errorf("creating msal cache root: %w", err)
 	}
 
+	authority := fmt.Sprintf("%s/%s", cDefaultAuthority, cDefaultRealm)
```

Comment:
NIT: you might consider calling rebuildMsalClient here as well, to remove some of the duplication.